### ISSUE TITLE
Control::has_point not listed as virtual method

### ISSUE THREAD

**Issue by bojidar-bg** [godotengine/godot#8018]
**Operating system or device - Godot version:**
Godot 2.1/HEAD, but it seems like master is somewhat affected too

**Issue description:**
While coding some stuff for a (non-FOSS) game, I named one of my methods as `has_point` on a `CanvasItem`-inheriting script that was assigned to some controls. I made it to `return true`, since that was to be overriden by other scripts... And then, suddenly, all the controls with that script started grabbing all clicks :laughing:

I decided to search the docs, since I suspected something like this might have happened, but `Control::has_point` wasn't listed, since it isn't bound at all.

Note that the `has_point` implementation on [`scene/gui/control.cpp:572-588`](https://github.com/godotengine/godot/blob/5dbf1809c6e3e905b94b8764e99491e608122261/scene/gui/control.cpp#L572-L588) calls the script method.

**Steps to reproduce:**

1. Make a script with:
```gdscript
extends Control
func has_point(point):
    return true
```
2. Add it to a Button node
3. Run
4. Profit (or actually, cry)

**Link to minimal example project:** Partly in a hurry here, so, maybe later on request.

**How to fix:**
1. Go to [`scene/gui/control.cpp:2516`](https://github.com/godotengine/godot/blob/5dbf1809c6e3e905b94b8764e99491e608122261/scene/gui/control.cpp#L2516)
2. Add a proper `BIND_VMETHOD` macro, like here: [`scene/main/node.cpp:2923`](https://github.com/godotengine/godot/blob/5dbf1809c6e3e905b94b8764e99491e608122261/scene/main/node.cpp#L2923)
```c++
BIND_VMETHOD(MethodInfo("_process", PropertyInfo(Variant::REAL, "delta")));
```
3. Compile, make sure that the docs list `Control::has_point(Vector2 point)`
4. Make a PR

**Comment by CrazyGuy108** [comment]
I'd like to try this as my first contribution.

**Comment by karroffel** [comment]
Also remove the regular `bind_method`? Otherwise the ClassDB entries might contain duplicates which mess stuff up. (aka #7960)

Not sure about that though

**Comment by akien-mga** [comment]
Fixed by #8023.
